pkg/kcp/wrappers.go

```diff
@@ -248,3 +272,47 @@ func generatePath(originalPath string, clusterPath logicalcluster.Path) string {
 	path += originalPath
 	return path
 }
+
+func cachedRESTMapper(config *rest.Config) (func(ctx context.Context) (meta.RESTMapper, error), error) {
+	httpClient, err := rest.HTTPClientFor(config)
+	if err != nil {
+		return nil, err
+	}
+
+	var mu sync.RWMutex
+	cache := expirable.NewLRU[logicalcluster.Path, meta.RESTMapper](1000, nil, time.Minute)
+	return func(ctx context.Context) (meta.RESTMapper, error) {
+		c, _ := kontext.ClusterFrom(ctx)
```

Comment:
The error from `kontext.ClusterFrom(ctx)` is ignored `(c, _ := kontext.ClusterFrom(ctx))`. We may consider it.

```
c, err := kontext.ClusterFrom(ctx)
if err != nil {
    // Handle error appropriately, perhaps returning a nil mapper with the error
    return nil, err
}
```

Reply:
Noticed that the second return value is a boolean, not an error. Will add a comment.

Reply:
done